importer/BeBruNl.py

```diff
@@ -0,0 +1,131 @@
+from Monument import Monument, Dataset
+import importer_utils as utils
+import importer as importer
+
+
+class BeBruNl(Monument):
+
+    def set_adm_location(self):
+        brussel_region = "Q240"
+        location = self.plaats
+        municip_dict = self.data_files["municipalities"]
+        if location:
+            location_match = [x for x in municip_dict if x[
+                "itemLabel"] == location]
+            if len(location_match) == 1:
+                self.add_statement("located_adm", location_match[0]["item"])
+        else:
+            self.add_statement("located_adm", brussel_region)
+
+    def set_heritage(self):
+        if self.has_non_empty_attribute("beschermd"):
+            prot_date = self.beschermd
+            try:
+                date_dict = utils.date_to_dict(prot_date, "%d/%m/%Y")
+                qualifier = {"start_time": {"time_value": date_dict}}
+                heritage = self.mapping["heritage"]["item"]
+                self.add_statement("heritage_status", heritage, qualifier)
+            except ValueError:
+                self.add_to_report("beschermd", self.beschermd)
+                return super().set_heritage()
+        else:
+            return super().set_heritage()
+
+    def set_address(self):
+        street_with_no = utils.remove_markup(self.adres.title()).rstrip(',')
+        whole_address = "{}, {}".format(street_with_no, self.plaats)
+        self.add_statement("located_street", whole_address)
+
+    def set_special_is(self):
+        objtype = self.objtype.lower()
+        types = self.data_files["type"]["mappings"]
+        special_types = [types[x]["items"]
+                         for x in types if x.lower() == objtype]
+        if len(special_types) > 0:
+            self.remove_statement("is")
+            for q in special_types[0]:
+                self.add_statement("is", q)
+
+    def set_style(self):
+        if self.has_non_empty_attribute("bouwstijl"):
+            style_raw = self.bouwstijl.lower()
+            styles = self.data_files["style"]["mappings"]
+            arch_style = [styles[x]["items"]
+                          for x in styles if x.lower() == style_raw]
+            if len(arch_style) > 0:
+                for q in arch_style[0]:
+                    self.add_statement("architectural_style", q)
+            else:
+                self.add_to_report(
+                    "bouwstijl", self.bouwstijl, "architectural_style")
+
+    def set_architect(self):
+        if self.has_non_empty_attribute("bouwdoor"):
+            if utils.count_wikilinks(self.bouwdoor) == 1:
+                arch_q = utils.q_from_first_wikilink("nl", self.bouwdoor)
+                self.add_statement("architect", arch_q)
+            else:
+                self.add_to_report("bouwdoor", self.bouwdoor, "architect")
+
+    def update_labels(self):
+        dutch = utils.remove_markup(self.omschrijving)
```

Comment:
Could you add the following test to test_importer_utils:TestWikitext to show that pipes will be handled correctly here.
```python


    def test_remove_markup_pipe(self):
        text = "[[Tegera Arena|Arenan]], huvudentrén"
        output = "Arenan, huvudentrén"
        self.assertEqual(utils.remove_markup(text), output)
```

Reply:
Done and it passes.